.github/workflows/scripts/check-collector-module-version.sh

```diff
@@ -49,8 +49,10 @@ check_collector_versions_correct() {
          version=$(echo "$mod_line" | cut -d" " -f2)
 
          # To account for a module on its own 'require' line,
-         # the version field is shifted right by 1
-         if [ "$version" == "$collector_module" ]; then
+         # the version field is shifted right by 1. Match
+         # with or without a trailing space at the end to account
+         # for the space at the end of some collector modules
+         if [ "$version" == "$collector_module" ] || [ "$version " == "$collector_module" ]; then
```

Comment:
Sounds like this should go on a different PR? Also cc @hickeyma

Reply:
I agree with @mx-psi. It's probably best put this in a separate PR for better trace afterwards of why the change. Do you want to push the PR @dashpole or if not, I can do it?

Reply:
Ok, I'll push fix shortly,

Reply:
Pushed #10885 for the fix.